.eslintrc.json

```diff
@@ -105,9 +105,14 @@
                 }
             }
         ],
-
-        // Todo: For each of these, investigate whether we want to enable them ✨
-        "@typescript-eslint/no-unused-vars": "off",
+        "@typescript-eslint/no-unused-vars": [
+            "error",
+            {
+                "argsIgnorePattern": "^_([^_]|_*$)",
```

Comment:
Hm, what is the second variant needed for? That regex is likely to be quite inefficient.

Reply:
- about 20 places in the codebase define arrow functions with no parameters, but use `_ => ... ` rather than `() => ...`. although now that I look at the rest of the codebase, there are many more places that use the normal `() => ...` form, so I could update those ~20ish places to match?
- These two places in `src/testRunner/parallel/shared.ts` do the same thing, but with two unused parameters. Therefore I made this generalized so that it supports a variable name made up of any number of underscores and nothing else
<img width="668" alt="image" src="https://github.com/microsoft/TypeScript/assets/7842684/116f59af-9592-4ea5-937e-2bbecb5513dd">

Reply:
Oh, then that can just be `^_($|[^_)`.

Reply:
Ignore me, I think what you write is fine; I misread it as a more complicated one that does arbitrary lookup.